### PR TITLE
Created header and main

### PR DESCRIPTION
**Pull request by Wes-Isaac**

### CODE REVIEW

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
```markdown
Hi Isaac Wesego, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have a professional README. You may use [this](https://github.com/microverseinc/readme-template) template from Microverse. 
Please add a description to PR. Moreover please use all the files from [this](https://github.com/microverseinc/linters-config/tree/master/html-css) linter config.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

Invalid Code Review Request **does not count** into the code reviews limit._
```

File: README.md [hunk 1]
```diff
@@ -1 +1,60 @@
...
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+Portfolio: setup and mobile version skeleton
+
+> This project is a my portfolio setup for mobile device.
+
+![screenshot](./images/Screenshot.png)
+
+Additional description about the project and its features.
```
Comment (DanteAlonsoHT) [inline README.md]: - [ ] [OPTIONAL] For this and the next projects you should add a better description, just try to explain what you did, what is the main scope, what kind of technologies or techniques you used, and so on. (Add like 3 lines or more at least in your description in your README file 😉).

File: README.md [hunk 2]
```diff
@@ -1 +1,60 @@
...
+
+- HTML
+- CSS
+
+
+## Live Demo
+
+[Live Demo Link](https://livedemo.com)
+
+
```
Comment (DanteAlonsoHT) [inline README.md]: - [ ] If your Live Demo Link is not working, please remove this section from your repository or fix the link adding a valid/enable URL to see your project😉

File: README.md [hunk 3]
```diff
@@ -1 +1,60 @@
...
+Clone the repo to your local machine.
+
+git clone https://github.com/Wes-Isaac/Portfolio.git
+
+cd Portfolio
+
+
+## Authors
+
+👤 **Author1**
```
Comment (DanteAlonsoHT) [inline README.md]: - [ ] [OPTIONAL] Use your name in that section, check:

What you have:
```
👤 **Author1**
```
What you can add:
```
👤 **Your Name**
```

File: index.html [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+</head>
+<body>
+    
+    <header>
+        <nav>
+            <ul class="nav">
+                <!-- <li><a href="#">Portfolio</a></li>
+                <li><a href="#">About</a></li>
+                <li><a href="#">Contact</a></li> -->
+                <li class="welcome">Welcome</li>
```
Comment (DanteAlonsoHT) [inline index.html]: In this case, your logo "Welcome" should be inside in an `a` tag, for instance:
```
<ul>
  <li class="welcome"><a href="#">Welcome</a></li>
</ul>
```

- [ ]  Please, add your logo inside an `a` tag, Also, I recommend you do the same for your nav links such as: `<li><i class="fal fa-bars"></i></li>`

File: index.html [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+                <li><i class="fal fa-bars"></i></li>
+            </ul>
+        </nav>
+    </header>
+    <main>
+        <section class="headline">
+            <h2>Hey there. I’m Isaac.
```
Comment (DanteAlonsoHT) [inline index.html]: - [ ] [OPTIONAL] The most important header text should be `h1`, you can use it there or later 😉

File: index.html [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css" integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p" crossorigin="anonymous"/>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" media="screen and (max-width:500px) " href="mobile.css">
+</head>
+<body>
+    
+    <header>
+        <nav>
```
Comment (DanteAlonsoHT) [inline index.html:15]: - [ ] [OPTIONAL] It would be nice if you apply responsive design for your project 😉 

image

File: index.html [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+               <span class="primary">I’m a software developer</span> </h2>
+            <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don’t hestiate to contact me.</p>
+            <ul class="social">
+             
+                <li><a href="#"><img src="./images/twitter.jpg" alt=""></a></li>
+                <li><a href="#"><img src="./images/Linkedinicon.jpg" alt=""></a></li>
+                <li><a href="#"><img src="./images/medium.jpg" alt=""></a></li>
+                <li><a href="#"><img src="./images/github.jpg" alt=""></a></li>
+                <li><a href="#"><img src="./images/angellist.jpg" alt=""></a></li>
```
Comment (DanteAlonsoHT) [inline index.html]: - [ ] In order to use better practices for HTML code, you should add something inside `alt=""`, you can try something like that:
```
<li><a href="#"><img src="./images/twitter.jpg" alt="twitter"></a></li>
```

File: index.html [hunk 8]
```diff
@@ -0,0 +1,44 @@
...
+            </ul> 
+            <i class="fal 2x fa-chevron-circle-down fa-2x dropdown"></i>
```
Comment (DanteAlonsoHT) [inline index.html]: - [ ] This button should be a `button` or an `a` tag. Please add it. For instance:
```
<a href="a"><i class="fal 2x fa-chevron-circle-down fa-2x dropdown"></i></a>  
```

**Review by DanteAlonsoHT — CHANGES_REQUESTED** [review]
<h1> Status: Required Changes 🚫 </h1>

Hi @Wes-Isaac,

Amazing so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
👇 _Check the comments under the review._ 👇 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 9]
```diff
@@ -0,0 +1,44 @@
...
+    <link rel="stylesheet" media="screen and (max-width:500px) " href="mobile.css">
+</head>
+<body>
+    
+    <header>
+        <nav>
+            <ul class="nav">
+                <!-- <li><a href="#">Portfolio</a></li>
+                <li><a href="#">About</a></li>
+                <li><a href="#">Contact</a></li> -->
```
Comment (SajjadAhmad14) [inline index.html:19]: [ OPTIONAL ] Please remove commented code as it makes the code base unclean and inefficient.

File: README.md [hunk 10]
```diff
@@ -1 +1,56 @@
-# Portfolio
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
```
Comment (SajjadAhmad14) [inline README.md:1]: [ OPTIONAL ] PR should have a descriptive summary about the features implemented or the changes made.

**Review by SajjadAhmad14 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

### Hi Isaac Wesego, 
## Congratulations 🎉 Thank you for implementing changes suggested previously 🙏
# OPTIONAL BUT RECOMMENDED
- [ ] Please see inline comments.
## Happy Coding 👏👏👏
#### Please leave any questions or comments in the PR if you have any doubts or unclarity!
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**